recipes/multipers/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
You can remove this file since it's in the src distribution on `pypi` (assuming you switch to that as the source)